QtImageViewer/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include <QApplication>
+#include <QThread>
+#include <QTimer>
+
+#include "VideoProcessor.hpp"
+#include "VideoViewer.hpp"
+
+#warning "Qt natively supports the MJPEG pixel format; a decoder is required for other pixel formats"
+
+#include <memory>
+
+using namespace std;
+
+int main( int argc, char *argv[] )
+{
+    cout << "Qt natively supports the MJPEG pixel format; a decoder is required" <<
+            "for other incoming source pixel formats" << endl;
+
+    // Object required to execute Qt GUI functionality
+    QApplication app( argc, argv );
+
+    auto videoProcessor =
+            std::unique_ptr< VideoProcessor >( new VideoProcessor );
```

Comment:
Also just realized that the `std` namespace is already being used so can be dropped here.